pkg/cover/backend/dwarf.go

```diff
@@ -363,41 +392,97 @@ func readTextRanges(debugInfo *dwarf.Data, module *vminfo.KernelModule, pcFix pc
 		if ent.Tag != dwarf.TagCompileUnit {
 			return nil, nil, fmt.Errorf("found unexpected tag %v on top level", ent.Tag)
 		}
-		attrName := ent.Val(dwarf.AttrName)
-		if attrName == nil {
+		attrName, ok := ent.Val(dwarf.AttrName).(string)
```

Comment:
Are you sure the nil check is unnecessary under any circumstances?

Reply:
At least in the docs they recommend relying on type conversion:
https://pkg.go.dev/debug/dwarf#Entry.Val

I assume that if `.(string)` led to `ok=true`, then the string should not be `nil`?

Reply:
It will panic if the interface was nil.

Reply:
Removed the `nil` check from `if language != nil && language.(int64) == languageRust {`